packages/onboarding/src/Client.ts

```diff
@@ -0,0 +1,66 @@
+import axios, { AxiosInstance, AxiosResponse, Method } from 'axios'
+import qs from 'qs'
+import { OnboardingRecord, Maker, DEWI_ONBOARDING_API_BASE_URL } from './types'
+
+type Response<T>={
+  code: number
+  data: T | null
+  success: boolean
+  errorMessage?: string
+  errors?: Array<any>
+}
+
+export default class Client {
+  private axios!: AxiosInstance
+
+  constructor(baseURL: string = DEWI_ONBOARDING_API_BASE_URL) {
+    this.axios = axios.create({
+      baseURL,
+    })
+  }
+
+  async execute<T>(method:Method, path: string, params: Object = {}) {
+    const query = qs.stringify(params)
+    let url = path
+    if (method === 'GET' && query.length > 0) {
```

Comment:
could `GET` specific logic be handled in the `get` function? then there wouldn't be a need to add case logic here

Reply:
yeah, I like that.